packages/documentation-site/patternfly-docs/pages/featured-posts-data.js

```diff
@@ -8,14 +17,6 @@ export const featuredPostsData = {
     imageURL:
       "https://miro.medium.com/v2/resize:fit:600/format:webp/0*j9kT2OT7NJuKJRwX",
   },
-  post2: {
-    title: "Webpack module federation | Think twice before sharing a dependency",
-    author: "Martin Maroši",
-    length: "9 min read",
-    URL: "https://medium.com/@marvusm.mmi/webpack-module-federation-think-twice-before-sharing-a-dependency-18b3b0e352cb",
-    // Use resize:fit:600 in URL
-    imageURL: "https://miro.medium.com/v2/resize:fit:600/format:webp/1*wvPsa-Gq_sblqb51mCto7w.png",
-  },
   post3: {
```

Comment:
On main right now, we've removed the `Why did the chicken cross the road` article. In your PR preview, there are 5 articles being displayed causing the layout to shift. Could you remove that article from this file rather than adding a `post5` to the data array?